src/manifest.json

```diff
@@ -6,7 +6,7 @@
     "icons": {
         "128": "assets/icon-128.png"
     },
-    "content_security_policy": "script-src 'self'; object-src 'self'",
+    "content_security_policy": "script-src 'self' 'unsafe-eval'; object-src 'self'",
```

Comment:
👀

Reply:
Need a better dev flow :) I keep committing this accidentally

Reply:
yeah, I agree that this is annoying. I have this ignored on IDE level, but..